docs/user/adaptive-learning.rst

```diff
@@ -9,10 +9,13 @@ Adaptive Learning
 
 Overview
 --------
-Artemis allows students and instructors to define and track competencies, which are overarching learning objectives that tie together various lectures and exercises. Adaptive Learning in Artemis is centered around these competencies, allowing users to customize their learning experience and objectives.
+Artemis allows students and instructors to define and track competencies, which are overarching learning objectives that tie together various lectures and exercises. Adaptive Learning in Artemis is centered around these competencies, allowing users to customize their learning experience and objectives. Furthermore, Artemis is able to provide students with individualized learning paths based on these competencies and their relations.
+
+Competencies
+------------
 
 Competencies for Students
--------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^
 Competencies allow students to understand the learning goals of the course and measure their progress towards achieving them.
 
 In Artemis, a competency is considered mastered by a student when they completed all linked learning objects (progress equals 100%) and have an adequate confidence level (average score greater or equal to mastery threshold set by the instructor).
```

Comment:
Also change this to `equal to the mastery threshold set`, as it refers to a specific one